plugins/CoreHome/stylesheets/dataTable/_rowActions.less

```diff
@@ -13,6 +13,7 @@ table.dataTable .dataTableRowActions a {
   float: left;
   padding: 6px 4px 6px 0;
   margin: 0;
+  width: auto!important;
```

Comment:
otherwise the rowaction links would have `width: inherit` from another rule, which might cause problems.